frontend/wallet/src/render/components/WalletName.re

```diff
@@ -1,3 +1,10 @@
+let getName = (pubkey, addressBook) => {
```

Comment:
We were trying to move away from this kind of function iirc because normally the two representations need different styles. Maybe if we need this as a special case somewhere we can put it in that location?